flytekit/annotated/launch_plan.py

```diff
@@ -228,3 +229,36 @@ def get_registerable_entity(self) -> SdkLaunchPlan:
             version=settings.version,
         )
         return self._registerable_entity
+
+
+class ReferenceLaunchPlan(ReferenceEntity, LaunchPlan):
+    def __init__(
+        self, project: str, domain: str, name: str, version: str, inputs: Dict[str, Type], outputs: Dict[str, Type]
```

Comment:
should we make the inputs / outputs as `interface`? and eventially `interface=auto` could be a supported thing?

Reply:
We can but then it's one more thing for the user to have to import in addition to `kwtypes` and the `Interface` object is kinda an internal detail right?  I think we can support automatic in the future by just not specifying inputs/outputs.  Either way.

Reply:
ok, what have we done for hive etc, lets keep it the same

Reply:
yeah hive follows inputs/outputs.